cmd/describe.go

```diff
@@ -0,0 +1,70 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/redhat-developer/ocdev/pkg/application"
+	"github.com/redhat-developer/ocdev/pkg/component"
+	"github.com/redhat-developer/ocdev/pkg/occlient"
+	"github.com/redhat-developer/ocdev/pkg/project"
+	"github.com/redhat-developer/ocdev/pkg/storage"
+	"github.com/redhat-developer/ocdev/pkg/url"
+	"github.com/spf13/cobra"
+)
+
+var describeCmd = &cobra.Command{
+	Use:   "describe [component_name]",
+	Short: "describe the given component",
+	Run: func(cmd *cobra.Command, args []string) {
+		client := getOcClient()
+
+		var cmpnt string
+		if len(args) == 0 {
+			var err error
+			cmpnt, err = component.GetCurrent(client)
+			checkError(err, "")
+		} else {
+			cmpnt = args[0]
+		}
+		source, urlList, storage := GetComponentDesc(client, cmpnt)
+		//Source Path
+		fmt.Println("Component", cmpnt, "with source in", source)
+		// URL
+		if len(urlList) != 0 {
+			fmt.Println("This Component is externally exposed via", urlList[0].URL)
+		}
+		//Storage
+		if len(storage) != 0 {
+			for _, store := range storage {
+				fmt.Println("This Component uses storage which is", store.Name, "of size", store.Size)
+			}
+		}
+
+	},
+}
+
+// Get Component Description
+func GetComponentDesc(client *occlient.Client, cmpnt string) (string, []url.URL, []storage.StorageInfo) {
```

Comment:
Wouldn't it make more sense to put this function to `component` pkg?

Reply:
```
go build -o ocdev ./main.go
import cycle not allowed
package main
	imports github.com/redhat-developer/ocdev/cmd
	imports github.com/redhat-developer/ocdev/pkg/component
	imports github.com/redhat-developer/ocdev/pkg/component
```

I get above error if I put this function in `/pkg/component`